frontend/packages/citizen-frontend/src/decisions/shared.tsx

```diff
@@ -2,16 +2,31 @@
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
-import { accentColors, greyscale } from '@evaka/lib-components/src/colors'
-import { faGavel, faCheck, faTimes } from '@evaka/lib-icons'
+import colors, {
+  accentColors,
+  blueColors,
+  greyscale
+} from '@evaka/lib-components/src/colors'
+import { faGavel, faCheck, faTimes, faEnvelope, faPlay } from '@evaka/lib-icons'
 import styled from 'styled-components'
 import { Decision, DecisionSummary } from '~decisions/types'
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core'
 
 export const Status = styled.span`
   text-transform: uppercase;
+  color: ${colors.greyscale.darkest}
+  font-size: 16px;
+  font-weight: 400;
+  padding-left: 0.5rem;
 `
 
-export const statusIcon = {
+export const statusIcon: {
+  [key: string]: { icon: IconDefinition; color: string }
+} = {
+  PROCESSING: {
```

Comment:
I don't think it's a good idea to mix decision list icons with application list icons.